operators/multiclusterobservability/manifests/base/grafana/dash-k8s-compute-resources-node-pods.yaml

```diff
@@ -490,7 +507,7 @@ data:
           "steppedLine": false,
           "targets": [
             {
-              "expr": "sum(container_memory_working_set_bytes{cluster=\"$cluster\", node=\"$node\", container!=\"\",job=\"kubelet\", metrics_path=\"\/metrics\/cadvisor\", image!=\"\"}) by (pod)",
+              "expr": "sum(container_memory_working_set_bytes{cluster=\"$cluster\", node=\"$node\", container!=\"\",job=\"kubelet\", metrics_path=\"/metrics/cadvisor\", image!=\"\"}) by (pod)",
```

Comment:
why modify expr here?

Reply:
This seems to be made when I updated the dashboard. Let me take a look at it.